sapientml_preprocess/templates/rename_columns.py.jinja

```diff
@@ -1,12 +1,16 @@
 # Remove special symbols that interfere with visualization and model training
 import re
 cols_has_symbols = {{ cols_has_symbols }}
-inhibited_symbol_pattern = re.compile(r"[\{\}\[\]\",:<'\\]+")
+rename_dict = {{ rename_dict }}
+inhibited_symbol_pattern = re.compile(r"[\{\}\[\]\",:<'\\\+]+")
+if len(rename_dict) == 0 :
+    rename_symbol_cols = {col: inhibited_symbol_pattern.sub("", col) if col in cols_has_symbols else col in cols_has_symbols for col in cols_has_symbols }
+else:
+    rename_symbol_cols = rename_dict
```

Comment:
We don't need show this conditional branch to users. Could you move it to "template's if statement"?

Reply:
Fixed conditional branching in template if statement.